### PR TITLE
57 postcode page

### PR DESCRIPTION
**Pull request by jamiejordanmt**

### CODE REVIEW

**Comment by jamiejordanmt** [comment]
[Content & Format Change - Postcode Page](https://trello.com/c/N4YZplJe/57-content-format-change-postcode-page)

**Review by Bhavesh-Patel — APPROVED** [review]

File: compoments/textInput.js [hunk 1]
```diff
@@ -63,31 +62,38 @@ class TextInput extends Component {
...
+  render() {
     return (
       <>
-        <h1 className="govuk-heading-l">{this.title}</h1>
-        <div className={this.state.error.msg ? 'govuk-form-group--error' : 'govuk-form-group'}>
-          <form action="" className={(this.long ? 'govuk-grid-column-two-thirds':'govuk-grid-column-one-third')+' govuk-!-padding-0'}>
+        <div
+          className={this.state.error.msg ? 'govuk-form-group--error' : 'govuk-form-group'}>
+          <form action=""
+            className={(this.long ? 'govuk-grid-column-two-thirds' : 'govuk-grid-column-one-third') + ' govuk-!-padding-0'}>
```
Comment (MK-NG) [inline compoments/textInput.js:71]: I'm trying to understand the `this.long` ternary operator? Is it to define the width of the text input? 

This should be done on the input with [fixed-width classes](https://design-system.service.gov.uk/components/text-input/#fixed-width-inputs)

Comment (jamiejordanmt) [reply compoments/textInput.js:71]: It's a boolean which is passed to the TextInput component, so this conditionally sets the class on this form. Currently, it is false on the postcode page and true on the contact person page, so the form should render with govuk-grid-column-one-third on the postcode page.

This css was already present and these are just formatting/spacing changes to the code. I think changes to this input (which also affects the contact-person page) is separate work.

**Comment by jamiejordanmt** [comment]
Closing this as it's well out of date.

**Pull request closed without merging**